src/app/clusters/content-launch-server/content-launch-server.cpp

```diff
@@ -219,7 +219,9 @@ bool emberAfContentLauncherClusterLaunchContentCallback(CommandHandler * command
     app::CommandResponseHelper<Commands::LaunchResponse::Type> responder(commandObj, commandPath);
 
     Delegate * delegate = GetDelegate(endpoint);
-    VerifyOrExit(isDelegateNull(delegate, endpoint) != true, err = CHIP_ERROR_INCORRECT_STATE);
+
+    VerifyOrExit(isDelegateNull(delegate, endpoint) != true && HasFeature(endpoint, ContentLauncherFeature::kContentSearch),
+                 err = CHIP_ERROR_INCORRECT_STATE);
     {
```

Comment:
@lazarkov This still has extra curly braces that don't seem to have a reason to exist....

Reply:
Sorry I haven't understood your comment at first, I'll remove it.